testnet/Stacks.toml

```diff
@@ -13,39 +13,40 @@ rpc_bind = "127.0.0.1:9000"
 ##    rpcuser=helium-node
 ##    rpcpassword=secret
 ##
-# [burnchain]
-# chain = "bitcoin"
-# mode = "helium"
-# peer_host = "127.0.0.1"
-# peer_port = 18444
-# rpc_port = 18443
-# rpc_ssl = false
-# username = "helium-node"
-# password = "secret"
-# timeout = 30
-# local_mining_public_key = "04ee0b1602eb18fef7986887a7e8769a30c9df981d33c8380d255edef003abdcd243a0eb74afdf6740e6c423e62aec631519a24cf5b1d62bf8a3e06ddc695dcb77"
-# burnchain_op_tx_fee = 1000
-# commit_anchor_block_within = 3000
+ # [burnchain]
+ # chain = "bitcoin"
+ # mode = "neon"
```

Comment:
We should keep an example of an helium config

Reply:
Yes, definitely, I ultimately shouldn't have committed changes to this file, I think. If anything, we should probably create a new Stacks.toml that's like `NeonLeader.toml`

Reply:
Addressed in f8674b907